style.css

```diff
@@ -1,2 +1,86 @@
+.credit-card  {
+    width: 500px;
+    height: 325px;
+    background-color: royalblue;
+    border-radius: 20px 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    font-family: "Geologica", sans-serif;
+    font-weight: 300;
+    font-optical-sizing: auto;
```

Comment:
Why `font-optical-sizing`?

Reply:
oh i copied it and rest code from the google fonts. it isn't necessary of course. I will delete it.